client/.editorconfig

```diff
@@ -0,0 +1,34 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
```

Comment:
This is interesting. I've never heard of this before but sounds very useful. There are plugins for a lot of editors.